po/common/common.pot

```diff
@@ -2802,7 +2802,7 @@ msgid ""
 msgstr ""
 
 msgctxt "title_separator"
-msgid " — "
+msgid " - "
```

Comment:
I'm a bit afraid of making punctuation translatable in fact, I don't really see how someone could translate " - " to a different dash. It might make sense for non roman alphabets, but for European languages we are going to get a random collection of translations...

So my intent was to get the smaller minus dash, because the long dash really looked ugly (in my opinion of course, I'm not an expert at all, and I have my own tastes :-) ) :

![image](https://user-images.githubusercontent.com/8158668/49384701-d71d4f00-f71b-11e8-93d7-4e89e25e382f.png)

I'm ok with the slighly smaller dash: – if we want to avoid the minus -. 

But to force it, we have to change the msgid I think, we can't leave the huge dash as the msgid.